src/lava/magma/runtime/runtime_services/runtime_service.py

```diff
@@ -295,3 +261,103 @@ def run(self):
                     if not enum_equal(rsp, MGMT_RESPONSE.DONE):
                         raise ValueError(f"Wrong Response Received : {rsp}")
                 self.service_to_runtime.send(MGMT_RESPONSE.DONE)
+
+
+class NxSDKRuntimeService(NcRuntimeService):
+    """NxSDK RuntimeService that implements NxCore SyncProtocol.
```

Comment:
There is no NxCore SyncProtocol. Should be rewritten after AbstractRuntimeService docstring is written.
A RuntimeService is a service process coordinating communication and execution within one SyncDomain implementing a particular SyncProtocol for all connected ProcModels.
Just like the PyLoihiRuntimeService, this one is also implementing the Loihi protocol. NxCore is just an API to configure and manage a Loihi system.

Reply:
Seems like there is lots of code duplication in this class with other RuntimeServices.